### PR TITLE
Sticky timeline sidebar

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

**Comment by samuveth** [comment]
Still missing some things for mobile

File: src/views/Timeline.vue [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+              v-text="$t('allSpaces')"
+              class="d-block px-4 py-2 sidenav-item"
+            />
+          </div>
+        </Block>
+      </div>
+    </div>
+    <div
+      class="col-12 col-lg-9 float-left pl-0 pl-lg-5"
+      style="margin-left: 240px"
```
Comment (bonustrack) [inline src/views/Timeline.vue:24]: You could use float-right instead of having this margin left

Comment (samuveth) [reply src/views/Timeline.vue]: Ok, let me try that

Comment (samuveth) [reply src/views/Timeline.vue]: Works well

File: src/views/Timeline.vue [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+            <router-link
+              :to="{ name: 'timeline', params: { scope: 'all' } }"
+              v-text="$t('allSpaces')"
+              class="d-block px-4 py-2 sidenav-item"
+            />
+          </div>
+        </Block>
+      </div>
+    </div>
+    <div class="col-12 col-lg-9 float-left pl-0 pl-lg-5 float-right">
```
Comment (bonustrack) [inline src/views/Timeline.vue:22]: You don't need `float-left` anymore if it work with `float-right`, also i think you can use Layout component directly instead.

Comment (bonustrack) [reply src/views/Timeline.vue:22]: Use float-right by default in Layout component and just add `<div style="position: fixed; width: 240px">` in the Timeline component

**Review by bonustrack — APPROVED** [review]
Great!

**Pull request merged**